osm2streets/src/render/intersection_markings.rs

```diff
@@ -15,6 +15,25 @@ impl StreetNetwork {
                 f.set_property("type", "sidewalk corner");
                 features.push(f);
             }
+
+            if let Some(ref crossing) = intersection.crossing {
+                match crossing.kind {
+                    CrossingKind::Signalized | CrossingKind::Marked => {
+                        for polygon in draw_zebra_crossing(self, intersection) {
+                            let mut f = Feature::from(polygon.to_geojson(Some(&self.gps_bounds)));
+                            f.set_property("type", "marked crossing line");
+                            features.push(f);
+                        }
+                    }
+                    CrossingKind::Unmarked => {
+                        for polygon in draw_unmarked_crossing(self, intersection) {
```

Comment:
Ideas for rendering styles welcome, especially for unmarked crossings -- by definition it's quite unclear how to indicate them! Of course these are going to be region-dependent; I'm starting with something simple and we can add configurability later (along with region-specific lane colors, default widths)

Reply:
Probably just a void with no road markings would suffice, unless there are regions where it’s normal for a centerline to continue uninterrupted past an unmarked crossing.

Reply:
Tough to do today, because osm2streets doesn't know how to continue any road markings into the "intersection polygon" area that gets calculated. That's something @BudgieInWA was working on remedying by detecting "intersections" that're really just merges / forks. I'll keep it in mind as an idea, thank you!